readme.md

```diff
@@ -50,14 +52,23 @@ You can control the maven build with the following profiles:
 * macm1
 
 ### Examples
-    mvn clean verify -Pwin64,latest,NOSDK -DskipTests=true (builds for latest eclipse and windows bits)
-    mvn clean verify -Plinux32,latest.NOSDK -DskipTests=true (builds for latest eclipse and linux 32 bits)
-    mvn clean verify -PSDK,win64,latest -DskipTests=true (builds the Sloeber SDK. For Sloeber programmers.)
-    
-To build for latest and the platform you are running on:
-
-    mvn clean verify -DskipTests=true
 
+* Build the latest version for the platform you are running on:
+
+    `mvn clean verify -DskipTests=true`
```

Comment:
NOSDK is best here
The SDK is an option to build a eclipse instance to do java development on sloeber